package-res/resources/web/pentaho/visual/base/types/selectionModes.js

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define(function() {
+  "use strict";
+
+  /**
+   * The `selectionModes` namespace contains a collection of methods that represent different ways
```

Comment:
@davidmsantos90 
The `selectionModes` namespace contains a collection of methods that represent different ways
to combine the current selection filter with the selection filter that was created by the user interaction with a visualization.